dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -270,7 +270,7 @@ def stage_extras
     end
 
     @stage_extras = {
-      next_stage_number: @lesson.next_level_number_for_lesson_extras(user),
+      next_lesson_number: @lesson.next_level_number_for_lesson_extras(user),
       stage_number: @lesson.relative_position,
```

Comment:
sorry I missed that the file I was trying to comment on is already in this PR. I could be misreading, but it looks like you  might need to update stage_number here too for this to not break

Reply:
good catch. I updated it.